homeassistant/components/light/mqtt.py

```diff
@@ -250,10 +253,13 @@ def brightness_received(topic, payload, qos):
             self.async_schedule_update_ha_state()
 
         if self._topic[CONF_BRIGHTNESS_STATE_TOPIC] is not None:
-            yield from mqtt.async_subscribe(
+            await mqtt.async_subscribe(
                 self.hass, self._topic[CONF_BRIGHTNESS_STATE_TOPIC],
                 brightness_received, self._qos)
             self._brightness = 255
+        elif self._optimistic_brightness and last_state\
+                and last_state.attributes.get('brightness'):
```

Comment:
Please `ATTR_BRIGHTNESS` from `light/__init__.py`. And on other places where a const which is defined in `light/__init__.py` make sense.